deployment/helm/tensorwave70b/scalarlm/values.yaml

```diff
@@ -1,12 +1,12 @@
 image:
-  repository: tensorwave/scalarlm-amd
-  tag: latest
+  repository: sudnya/scalarlm-rocm
+  tag: v0.8
```

Comment:
I am happy to revert this file if you prefer @gdiamos . In my opinion, it is best to update this file on main once 70B is successfully deployed on the llama70b namespace.